10up/action-pr-helper
# Add support for Merge Conflict

### Description of the Change
PR adds support for the merge conflict. When a PR has merge conflicts, add a comment to the PR submitted asking them to refresh their PR and apply the `needs: refresh` label, Once the merge conflict is resolved it removes the comment and label.


Closes #43 

### How to test the Change
1. Add workflow to the repository using this action @ PR branch. (you can use any repo for the test)
1. Raise and dummy PR to the repo
1. Update the base branch to create a conflict in raised PR
1. Verify that label and comment for conflict added to PR
1. Resolve conflicts in PR
1. Verify that label and comment got removed from the PR 

### Changelog Entry
> Added - Support for Merge Conflict


### Credits
Props @iamdharmesh 


### Checklist:
